app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,9 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="dark_blue">#192841</color>
+    <color name="light_grey">#E7DEDE</color>
+    <color name="app_background">#D6E3EF</color>
+    <color name="blue">#6E85C8</color>
```

Comment:
Storing those in a constant is great. Do you think we should keep the name of the colours or define constants which would indicate what they're used for ?

Reply:
Once we define the final colors we could add the main ones to the themes file instead!